regression/validate-trace-xml-schema/check.sh

```diff
@@ -13,6 +13,7 @@ fi
 xmllint="$(command -v xmllint)"
```

Comment:
Also, for something unrelated to this PR, but why is `command -v` used here instead of `whereis`? Are there any benefits? 

I don't mind it, it was just surprising to me - never seen this before, and for this functionality, I've always seen `whereis` being used instead.

Reply:
`command` is very simple. It just checks if something is on `$PATH`, optionally returns the path to it and gives me an error code if can’t be found. `whereis` seems to be a more complicated tool used for actually locating things, which is not what I’m trying to do.